scripts/hooks/gh-pr-guard.sh

```
@@ -684,42 +700,157 @@ for j in "${!TOKENS[@]}"; do
+#
+# #171 / #170 retrospective: pre-this-change the hook fetched
+# only labels and let `gh pr merge` run even when GitHub's
+# `mergeStateStatus` was BLOCKED (failing CI / active
+# CHANGES_REQUESTED). A PR could merge with red CI on every
+# matrix cell because nothing in the merge path actually
+# blocked. The new check is defense-in-depth behind branch
+# protection: even if branch protection is misconfigured or
+# disabled for an emergency hotfix, the hook will still refuse
+# to dispatch the merge.
+GH_JQ='"\(.mergeStateStatus)|\([.labels[].name] | join(","))"'
+GH_ARGS=(pr view --json labels,mergeStateStatus --jq "$GH_JQ")
```

Shellcheck (0.11.0)

[warning] Use spaces, not commas, to separate array elements.

(SC2054)